requirements-freqai.txt

```diff
@@ -6,3 +6,9 @@ scikit-learn==1.1.2
 joblib==1.1.0
 catboost==1.0.6; platform_machine != 'aarch64'
 lightgbm==3.3.2
+torch==1.12.1
+stable-baselines3==1.6.0
+gym==0.21.0
+tensorboard==2.9.1
+optuna==2.10.1
+sb3-contrib==1.6.0
```

Comment:
i'm ... slightly sceptical about these requirements - they seem pretty heavy - and i'm not sure they'll install properly on all devices / architectures (see catboost ...).

Reply:
The only one that doesn't have any business there is optuna. 

Yes, we need to test the rest on other architectures. Hoping to get some beta testers from various architectures via this PR.

Reply:
Also, maybe it is better for us to have RL be a "sub install?" 

User wants to install FreqAI, so says Yes. Next question, does the user wish to install RL dependencies too? 

Thoughts?

Reply:
probably - but i'm hesitant to have 200 requirements files for different needs - you'll soon end in "why is this not working" due to partial installs - when needs change, or users feel the need to "test that new cool model" ...